README.md

```diff
@@ -101,8 +101,21 @@ the system is intentionally less aggressive about automatically applying those
 (as can be seen in the example above).
 
 Sentence entries also contain a list of `tokens`. The tokens
-originate in the [Tokenizer package](https://github.com/mideind/Tokenizer)
-and their format is documented there.
+originate in the [Tokenizer package](https://github.com/mideind/Tokenizer) and contain the following fields:
+
+`i`: Character index of token start.
+`k`: Token type number, (WORD, DATEREL, AMOUNT, etc.). Mapping from numbers to strings is found in the documentation for the [Tokenizer package](https://github.com/mideind/Tokenizer).
```

Comment:
Number identifying the token type (WORD, DATEREL, AMOUNT, etc.). The mapping from numbers to token types can be found in the documentation for the [Tokenizer package](https://github.com/mideind/Tokenizer).

Reply:
Done